src/index.ts

```diff
@@ -123,7 +121,7 @@ export class CdktfProviderProject extends cdk.JsiiProject {
       license: "MPL-2.0",
       releaseToNpm: true,
       minNodeVersion,
-      devDeps: [`@cdktf/provider-project@^${version}`, "dot-prop@^5.2.0"],
+      devDeps: ["dot-prop@^5.2.0", ...(options.devDeps ?? [])],
```

Comment:
Quick question, as I haven't checked how Projen handles this, but does this now require us to set a peer dependency on the provider project in the `projenrc.js` file that sets this up?
If yes, we should update our tests to do so, too. We also need to remember this change when we roll it out and change that for each of our pre-built providers as well (there's a single [template](https://github.com/cdktf/cdktf-repository-manager/blob/main/projenrc.template.js) file, that helps with that).

Reply:
I'll admit that my knowledge around peer dependencies is a bit thin. My gut is 'no', because you could always switch to a completely different project type - or even a fork of the came project type but with a different name. If the peer dependency doesn't complicate or prevent such changes, I could see it working. Otherwise, maybe it remains intentionally loose; at the very least you have everything in the same `.projenrc.js` file: the dependency _and_ the one time it gets imported.

Reply:
Yeah, setting the `devDependency` on the Projen project used seems like the right thing to do. Fyi: I pushed a commit that changes our tests to do so, too, to make it more explicit how this Projen project should be used.